README.md

```diff
@@ -54,20 +54,19 @@ import time
 import data_bridges_client
 from data_bridges_client.rest import ApiException
 from pprint import pprint
+from data_bridges_client.token import WfpApiToken
 
-# Defining the host is optional and defaults to https://api.wfp.org/vam-data-bridges/2.0.0
```

Comment:
This example code show how to use the `WfpApiToken` class from `token.py` to authenticate in the API.